src/configurations/destinations/iterable/ui-config.json

```diff
@@ -37,6 +37,26 @@
                         }
                       ]
                     }
+                  },
+                  {
+                    "type": "singleSelect",
+                    "label": "Data Center",
+                    "configKey": "dataCenter",
+                    "regex": "^(.{0,100})$",
+                    "regexErrorMessage": "Invalid Data Center",
+                    "options": [
+                      {
+                        "label": "USDC",
+                        "value": "USDC"
+                      },
+                      {
+                        "label": "EUDC",
+                        "value": "EUDC"
+                      }
+                    ],
+                    "required": true,
```

Comment:
`required` also is not valid in new form builder, all fields under `Initial setup` are considered required by the framework.